### PR TITLE
Revert "fix issue 19590 - `__traits allMembers` should put fully qual…

### PR DESCRIPTION
**Pull request by schveiguy**
…ified names for imports"

This reverts commit 801e841e7dc1cef01c17ee0df5adca29b8c1e1bf.

This can be re-added to master as a new PR and we can discuss there.

@NilsLankila

### CODE REVIEW

**Review by Geod24 — APPROVED** [review]
Since it's the most controversial change of this release, that seems sensible.

**Comment by UplinkCoder** [comment]
Yep we should deferr this to the next major.

**Comment by adamdruppe** [comment]
so fun fact: adrdox' internal system considers imports to be members too.... and yeah I don't like it. It is convenient in that it shows all the stuff in the namespace and gives me a decent list, but it is also just weird. Imports are a different conceptual beast altogether. Especially nested imports inside other aggregates that are sometimes relevant but still pretty universally need special treatment.

Of course, I haven't thought of how I want to fix it in there yet so there's still some quirks in generated pages because of it.... but at least there I can go `if(auto d = cast(ImportDecl) member) continue;` whereas in real D there's no such filter possible at this time. So maybe a filter is good enough but I kinda think longer term we should separate them out.

**Comment by UplinkCoder** [comment]
Agreed!

**Comment by wilzbach** [comment]
Reverts do not need the 72h waiting period. Our policy is to err on the side of caution and and almost always revert whenever it's clear that a potential problem has been uncovered that likely won't be resolved before the release is shipped.

tl;dr: don't feel shy to revert.

(to avoid merge conflicts, stable should be merged back into master, before this gets re-added to `master`).
